recipes/visidata/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "visidata" %}
+{% set version = "1.1" %}
+{% set sha256 = "ae8b62f78ca30fc14695813694fea1d8b571b4a45efd61ced1ce1681e8edf36e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<34]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python >=3.4
```

Comment:
I'm commenting here because it's easier. This is wrong. You should remove `>=3.4` from here and keep the skip.

Reply:
:+1: Will do so now.

Reply:
I should remove it from the `run` requirements as well?

Reply:
Yes